db/migrate/20170223114114_lock_seeded_templates.rb

```diff
@@ -0,0 +1,14 @@
+require Rails.root + 'db/seeds.d/07-data.rb'
+require Rails.root + 'db/seeds.d/08-data.rb'
+
+class LockSeededTemplates < ActiveRecord::Migration
+  def up
+    names = (SEEDED_TEMPLATES + SEEDED_PARTITION_TABLES).map { |attrs| attrs[:name] }
+    Template.where(:name => names).update_all(:locked => true)
```

Comment:
Why is this needed if `:locked => true` is set in the seed https://github.com/theforeman/foreman/pull/4320/files#diff-151633105c29141fe3aac7603de185a5R25 ? Is this only relevant for partition tables?